packages/mobile/src/screens/collection-screen/CollectionScreen.tsx

```diff
@@ -75,23 +73,11 @@ const useStyles = makeStyles(({ spacing }) => ({
  */
 export const CollectionScreen = () => {
   const { params } = useRoute<'Collection'>()
-
-  // params is incorrectly typed and can sometimes be undefined
-  const { id = null, searchCollection, collectionType } = params ?? {}
-
-  const cachedCollection = useSelector((state) =>
-    getCollection(state, { id })
-  ) as Nullable<Collection>
-
-  const cachedUser = useSelector((state) =>
-    getUser(state, { id: cachedCollection?.playlist_owner_id })
-  )
-
-  const collection = cachedCollection ?? searchCollection
-  const user = cachedUser ?? searchCollection?.user
+  const { data: collection } = useCollectionByParams(params)
+  const { data: user } = useUser(collection?.playlist_owner_id)
```

Comment:
so nice to be able to chain these without passing enabled. Very happy with this pattern

Reply:
I actually think I'll grab the user from the collection data though I'm realizing for this component

Reply:
ooh good point. Is that an anti-pattern now?

Ties into discussions about normalization and what we store in cache. TBD

Reply:
yeah i wonder what the move should be here... im not sure yet, so ill just use it since it's easy. our updates to do cover these entities fwiw